src/store/preferences.ts

```diff
@@ -151,14 +159,25 @@ export const actions = {
         body: JSON.stringify(state.preferences)
       });
 
+    if (!result.ok) {
+      const severities = { ...state.severities, error: true };
+
+      commit('SET_SEVERITIES', severities);
+      commit('SET_PREFERENCES_ERROR', await result.text());
+
+      return severities;
+    }
+
     const changes: Record<string, {severity: 'reset' | 'restart'}> = await result.json();
     const values = Object.values(changes).map(v => v.severity);
     const severities = {
       reset:   values.includes('reset'),
       restart: values.includes('restart'),
+      error:   false
```

Comment:
Prefer trailing commas to make future change slightly cleaner (but if you don't feel like it then that's fine too)

Reply:
I've never found this to be too much of a problem for me, but I'm of the opinion that consistency is king.

If this is something that we'd like to enforce, I think it makes sense to update our eslint rule to make it consistent across the board. Raised PR #2703 so we can discuss more in-depth as a group 🙂